### ISSUE TITLE
Follower and Following count is 0 on own profile page, even though there should be more

### ISSUE THREAD

**Issue by anarchist912** [pixelfed/pixelfed#3613]
I had this for quite a while, so I thought I'd report it now.

It shows 0 for Followers and Following, but as you can see in the screenshot, it should probably display about 4 for followers:

As you can see, the follower count comes like that from the endpoint for the account:

```JSON
{
   ...
  "display_name": "",
  "discoverable": true,
  "locked": false,
  "followers_count": 0,
  "following_count": 0,
  "statuses_count": 7,
  ...
  "url": "https:\\/\\/pixelfed.social\\/ssh",
  "avatar": "https:\\/\\/pixelfed.social\\/storage\\/avatars\\/031\\/884\\/975\\/557\\/902\\/745\\/6\\/41hyhbnvBOTPcE2pXQ9f_avatar.jpg?v=2",
  "website": "https:\\/\\/bandcamp.com",
  "local": true,
  "is_admin": false,
  "created_at": "2021-07-08T20:35:05.000000Z",
  "header_bg": null,
  "last_fetched_at": null,
  "pronouns": [],
  "location": null
}
```
While the followers endpoint responds with this:
```JSON
[
  {
    "id": "443549274445968074",
    "username": "drevny.ritm",
    "acct": "drevny.ritm",
    "display_name": "\\u0414\\u0440\\u0435\\u0432\\u043d\\u0438\\u0439 \\u0420\\u0438\\u0442\\u043c \\ud83c\\udfbc",
    "discoverable": true,
    "locked": false,
    "followers_count": 16,
    "following_count": 71,
    "statuses_count": 6,
    "note": "Psychedelic rock\nDubrovenka chan: <a href=\"https:\\/\\/youtube.com\\/channel\\/UCdOCOvy4bHZjXl-WzIe4_BQ\" rel=\"external nofollow noopener\" target=\"_blank\">https:\\/\\/youtube.com\\/channel\\/UCdOCOvy4bHZjXl-WzIe4_BQ<\\/a>\nVids: <a href=\"https:\\/\\/www.youtube.com\\/channel\\/UCmUbeY18vSJatCYWauFw4Uw\\/featured\" rel=\"external nofollow noopener\" target=\"_blank\">https:\\/\\/www.youtube.com\\/channel\\/UCmUbeY18vSJatCYWauFw4Uw\\/featured<\\/a>\nReleases: \nhttps:\\/\\/drevnyritm.bandcamp.com\\/releases\nOther: https:\\/\\/band.link\\/np3q1",
    "note_text": "Psychedelic rock\nDubrovenka chan: https:\\/\\/youtube.com\\/channel\\/UCdOCOvy4bHZjXl-WzIe4_BQ\nVids: https:\\/\\/www.youtube.com\\/channel\\/UCmUbeY18vSJatCYWauFw4Uw\\/featured\nReleases: \nhttps:\\/\\/drevnyritm.bandcamp.com\\/releases\nOther: https:\\/\\/band.link\\/np3q1",
    "url": "https:\\/\\/pixelfed.social\\/drevny.ritm",
    "avatar": "https:\\/\\/pixelfed.social\\/storage\\/avatars\\/044\\/354\\/927\\/444\\/596\\/807\\/4\\/eP6B6cg8iEgc8AF5MK4g_avatar.jpg?v=4",
    "website": "https:\\/\\/vk.com\\/club193656283",
    "local": true,
    "is_admin": false,
    "created_at": "2022-06-17T23:06:27.000000Z",
    "header_bg": null,
    "last_fetched_at": null,
    "pronouns": [],
    "location": null
  },
  {
    "id": "411835392159414639",
    "username": "bendy",
    "acct": "bendy",
    "display_name": "BENDY",
    "discoverable": true,
    "locked": false,
    "followers_count": 9,
    "following_count": 12,
    "statuses_count": 2,
    "note": "Made music, might delete later\n\nNew single New Skin available \\ud83d\\udc47 <a href=\"https:\\/\\/open.spotify.com\\/album\\/6fEBTFXjiNYUjfrLhqP0Mj?si=f3FZY26OQVmryFb1rbJ52w\" rel=\"external nofollow noopener\" target=\"_blank\">https:\\/\\/open.spotify.com\\/album\\/6fEBTFXjiNYUjfrLhqP0Mj?si=f3FZY26OQVmryFb1rbJ52w<\\/a> (also available on Soundcloud <a href=\"https:\\/\\/soundcloud.com\\/sloppybendy\" rel=\"external nofollow noopener\" target=\"_blank\">https:\\/\\/soundcloud.com\\/sloppybendy<\\/a> and other platforms)",
    "note_text": "Made music, might delete later\n\nNew single New Skin available \\ud83d\\udc47 https:\\/\\/open.spotify.com\\/album\\/6fEBTFXjiNYUjfrLhqP0Mj?si=f3FZY26OQVmryFb1rbJ52w (also available on Soundcloud https:\\/\\/soundcloud.com\\/sloppybendy and other platforms)",
    "url": "https:\\/\\/pixelfed.social\\/bendy",
    "avatar": "https:\\/\\/pixelfed.social\\/storage\\/avatars\\/041\\/183\\/539\\/215\\/941\\/463\\/9\\/CBBT4EDdmFmGMUAidFl6_avatar.png?v=2",
    "website": "https:\\/\\/open.spotify.com\\/artist\\/1aWBhY8XJgHH0o9YRLscBP?si=vcqqR12xRviuBOhSd04GHg",
    "local": true,
    "is_admin": false,
    "created_at": "2022-03-22T10:46:49.000000Z",
    "header_bg": null,
    "last_fetched_at": null,
    "pronouns": [],
    "location": null
  },
  {
    "id": "1646",
    "username": "cedric",
    "acct": "cedric",
    "display_name": "Cedric",
    "discoverable": true,
    "locked": false,
    "followers_count": 215,
    "following_count": 185,
    "statuses_count": 193,
    "note": "Photos under CC BY-SA 4.0 license.",
    "note_text": "Photos under CC BY-SA 4.0 license.",
    "url": "https:\\/\\/pixelfed.social\\/cedric",
    "avatar": "https:\\/\\/pixelfed.social\\/storage\\/avatars\\/000\\/000\\/001\\/646\\/avatar.jpeg?v=2",
    "website": "https:\\/\\/fosstodon.org\\/@cedric",
    "local": true,
    "is_admin": false,
    "created_at": "2018-08-15T17:56:21.000000Z",
    "header_bg": "default",
    "last_fetched_at": null,
    "pronouns": [],
    "location": null
  },
  {
    "id": "318681788945928192",
    "username": "juusan",
    "acct": "juusan",
    "display_name": "juusan",
    "discoverable": true,
    "locked": false,
    "followers_count": 25,
    "following_count": 21,
    "statuses_count": 33,
    "note": "i don't know who i am yet and i'm not saying anything until i do",
    "note_text": "i don't know who i am yet and i'm not saying anything until i do",
    "url": "https:\\/\\/pixelfed.social\\/juusan",
    "avatar": "https:\\/\\/pixelfed.social\\/storage\\/avatars\\/031\\/868\\/178\\/894\\/592\\/819\\/2\\/3DnlTGcP54OY9OMD4CQK_avatar.jpg?v=2",
    "website": null,
    "local": true,
    "is_admin": false,
    "created_at": "2021-07-08T09:27:39.000000Z",
    "header_bg": null,
    "last_fetched_at": null,
    "pronouns": [],
    "location": null
  }
]
```

**Comment by MrAn0nym** [comment]
Most likely related to #3539

**Comment by anarchist912** [comment]
Sorry, my bad, we can close this.

I had a look at the code in the Profile.php and saw that 0 is returned when showing the followers/following count is disabled in the profile settings:

```php
if($this->domain == null && $this->user->settings->show_profile_follower_count == false) {
  return 0;
}
```

I changed my settings and now it's getting displayed correctly. I forgot that I had disabled showing the follower count in the past and didn't even remember that this feature existed.
